lib/data_update_scripts/20200612173137_backfill_last_active_at_for_broadcasts.rb

```diff
@@ -0,0 +1,15 @@
+module DataUpdateScripts
+  class BackfillLastActiveAtForBroadcasts
+    def run
+      # Broadcasts didn't have a last_active_at column until
+      # 20200609195523_add_last_active_at_to_broadcasts was created, thus we can
+      # safely assume that only those with `last_active_at IS NULL` need their
+      # timestamps overridden.
+      # This updates the Broadcast's last_active_at to be the date and time
+      # in which the Broadcast was last updated.
+      Broadcast.where(last_active_at: nil).order(title: :asc).each do |cast|
+        cast.update(last_active_at: cast.updated_at)
```

Comment:
Please see my "post-deployment tasks" comment in the PR description regarding this script!

Reply:
Hm, I have some doubts around this. Why do we need to backfill the data for pre-existing broadcasts?

The `last_active_at` column isn't _required_ and we allow that column to be `null` in [the migration](https://github.com/thepracticaldev/dev.to/pull/8451/files#diff-59c9ac156b94b4411f337406d7e5da9aR3). If we run a script to backfill previous broadcast data to whenever they were _updated_, we're setting an attribute manually which in the future will lead someone to believe that the broadcast was _at some point_, set to `active`. This is will not necessarily be true. For example, if there is a broadcast on production that has never been set to active, but has been updated/saved at some point, when we ran this script, we'd now actually be mutating the data attached to it and effectively saying that "this broadcast was once active", even though it never was. The truth is that the broadcast was _updated_ at some point, but that may not have anything to do with whether or not it was active :)

TL;DR: we really shouldn't ever set `last_active_at` manually since this will cause potential confusion (and maybe bugs) down the road; it should only be set when the broadcast was _actually_ activated.

Reply:
OH! I thought (after looking at Broadcasts) that this would be necessary since all of the Broadcasts in production will not have a `last_active_at` timestamp by default, but I see what you mean - I overoptimized here and since removed this file.